adafruit_ssd1306.py

```diff
@@ -35,6 +35,19 @@ class _SSD1306:
     """Base class for SSD1306 display driver"""
     def __init__(self, framebuffer, width, height, external_vcc):
         self.framebuf = framebuffer
+        # note this is a breaking change:
+        # previously these functions were wrapped
+        # with a default color of "1"
```

Comment:
This comment should go in the PR message not the code. I bet col=1 below is actually column not color too.